make/tools.mk

```diff
@@ -109,6 +109,10 @@ ifdef WINDOWS
 	$(V1) ./downloads/qt-opensource-windows-x86-mingw492-5.5.0.exe
 endif
 
+	# Configure Qt Creator user project files (GCS and flight firmware)
```

Comment:
well you might want want to clobber your configuration as part of installing the  SDK. what if you had customized it and this overwrote it? it's better to have small granular targets that do what they say